css/layout.css

```diff
@@ -1,151 +1,107 @@
-*{
-  border: 1px solid black;
-  text-align: center;
-}
 #everything {
-  width: 960px;
-  height: 768px;
-  margin: auto;
-}
-
-#a {
-  width: 960px;
-  height: 100px;
-  line-height: 100px;
-}
-
-#bc{
   margin: auto;
-  height: 600px;
-  width: 960px;
-  overflow: hidden;
-}
-
-#b{
-  width: 200px;
-  height: 600px;
-  float: left;
-  line-height: 600px;
-}
-
-#everythingelse{
-  width: 760px;
-  height: 600px;
-  float: left;
+  display: grid;
+  height: 700px;
+  max-width: 960px;
+  grid-template-rows: 1fr 6fr; 
+  text-align: center;
 }
 
-#cd{
-  margin: auto;
-  height: 200px;
-  width: 760px;
-  overflow: hidden;
+#bc {
+  display: grid;
+  grid-template-columns: 200px 760px;
 }
 
-#c{
-  height: 200px;
-  width: 380px;
-  float: left;
-  line-height: 200px;
+#everythingelse {
+  display: grid;
+  grid-template-rows: 200px 250px 150px;
+  grid-template-rows: 33.33% 41.66% 25%;
 }
-
-#d{
-  height: 200px;
-  width: 380px;
-  float: left;
-  line-height: 200px;
+#cd {
+  display: grid;
+  grid-template-columns: 50% 50%;
 }
-
 #efgh{
-  margin: auto;
-  height: 250px;
-  width: 760px;
-  overflow: hidden;
+  display: grid;
+  grid-template-columns: 25% 25% 25% 25%;
 }
 
-#e{
-  height: 250px;
-  width: 190px;
-  float: left;
-  line-height: 250px;
+#a {
+  line-height: 100px;
 }
-
-#f{
-  height: 250px;
-  width: 190px;
-  float: left;
-  line-height: 250px;
+#b {
+  line-height: 600px;
 }
-
-#g{
-  height: 250px;
-  width: 190px;
-  float: left;
-  line-height: 250px;
+#c, #d {
+  line-height: 200px;
 }
-
-#h{
-  height: 250px;
-  width: 190px;
-  float: left;
+#e, #f, #g, #h {
   line-height: 250px;
 }
-
-#i{
-  height: 150px;
-  width: 760px;
-  float: left;
+#i {
   line-height: 150px;
 }
-
 @media only screen and (max-width: 767px) {
   #everything {
-    width: 480px;
+    display: grid;
+    max-width: 480px;
     height: 1060px;
+    grid-template-rows: 75px 150px 125px 125px 250px 250px 80px;
+    /*grid-template-columns: 1fr;*/
   }
   
   #a {
     width: 480px;
     height: 75px;
     line-height: 75px;
   }
-  
+  #bc {  
+    width: 480px;
+  }
+  /*
   #bc{
-    margin: auto;
+    display: grid;
+    grid-row-start: 2;
+    grid-template-rows: 150px 830px;
     width: 480px;
-    height: 935px;
   }
-  
+  */
   #b{
+    display: grid;
+    grid-row: 2 / 3;
     width: 480px;
     height: 150px;
     line-height: 150px;
   }
-  
+  /*
   #everythingelse{
-    background-color: #ccc;
+    grid-row-start: 3;
+    display: grid;
     width: 480px;
     height: 835px;
   }
 
   #cd{
-    margin: auto;
-    height: 200px;
-    width: 760px;
-    overflow: hidden;
-  }
-  
+    display: grid;
+    height: 250px;
+    width: 480px;
+    }
+  */
   #c{
+    display: grid;
+    grid-row: 3 / 4;
     height: 125px;
     width: 480px;
     line-height: 125px;
   }
 
   #d{
+    grid-row: 4 / 5;
     height: 125px;
     width: 480px;
     line-height: 125px;
   }
-  
+  /*
```

Comment:
this is a a lot of dead code, you'll want to get to the point of removing everything you're not using when you submit.